packages/lesswrong/lib/routes.ts

```diff
@@ -733,16 +732,15 @@ const forumSpecificRoutes = forumSelect<Route[]>({
       path: `/${taggingNamePluralSetting.get()}/:slug/subforum`,
       redirect: (routerLocation: RouterLocation) => {
         const { params: {slug}, query, hash } = routerLocation
-        const isRouteSubforum = subforumSlugsSetting.get().includes(slug)
 
         const redirectQuery = pickBy({
           ...query,
           tab: "subforum",
           commentId: query.commentId || hash?.slice(1)
         }, v => v)
 
-        // If the route is not declared as a subforum but somehow the user has clicked on a subforum link, redirect to the /subforum2 path, which will always display like a subforum
-        return `/${taggingNamePluralSetting.get()}/${slug}${isRouteSubforum ? '' : '/subforum2'}?${qs.stringify(redirectQuery)}${hash}`
+        // Redirect to the /subforum2 path, which will always display like a subforum regardless of whether isSubforum is true
+        return `/${taggingNamePluralSetting.get()}/${slug}/subforum2?${qs.stringify(redirectQuery)}${hash}`
```

Comment:
Why are we continuing to support `/subforum2`? We don't actually need it anymore right?

Reply:
It's kind of useful for seeing how a subforum will look in prod before actually launching it, also the subforum page is the easiest place to add subtags (which is a bit of a hack). These aren't great reasons but seeing as it's not linked from anywhere I decided it was worth leaving